test/energy_scan/mult_and_spectra_plotter.py

```diff
@@ -302,7 +302,10 @@ def sqrts(elab):
                     energy_elab = float(exp_file.split(i)[1].split('_')[0])
                     energy = sqrts(energy_elab) if (i != 'ecm') else energy_elab
                     break
-            energy = np.trunc(energy * 1.e2) / 1.e2
+            if(energy<13):
```

Comment:
I guess that the value _13_ is simply a choice determined by the appearance of the plot.
It is not a fundamental recommendation (i.e. you are free to ignore it), but I would suggest to transform it into a parameter (defined inside the function) and add a comment about why you introduced it. It might be helpful to future developers (albeit I do not expect many changes to this function for a long time, if ever).

Reply:
Good, thank you!